controller/network/network.go

```diff
@@ -754,7 +754,7 @@ func sendRoute(r *Router, createMsg *ctrl_pb.Route) (xt.PeerData, error) {
 		}
 		return nil, fmt.Errorf("unexpected response type %v received in reply to route request", msg.ContentType)
 
```

Comment:
Do we want to log what the timeout was on error, or just have people look it up in the controller config?

Reply:
Implemented.